packages/bot-web-ui/src/utils/__tests__/window-size.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import {
+    getMainContentHeight,
+    getMainContentWidth,
+    setMainContentWidth,
+    setInnerHeightToVariable,
+} from '../window-size';
+
+describe('DOM utilities of window-size file', () => {
+    beforeEach(() => {
+        render(<div className='bot' data-testid='mock-bot' />);
+    });
+
+    it('Should get main content height', () => {
+        const bot_element = screen.getByTestId('mock-bot');
+
+        if (!bot_element) {
+            throw new Error("Element with class '.bot' was not found");
+        }
+        bot_element.style.setProperty('--bot-content-height', '100px');
+
+        expect(getMainContentHeight()).toBe('100px');
+    });
+
+    it('Should get main content width', () => {
+        const bot_element = screen.getByTestId('mock-bot');
+
+        if (!bot_element) {
+            throw new Error("Element with class '.bot' was not found");
+        }
```

Comment:
Same as the above question for this condition :raised_hand:

Reply:
done✅